web/edit/server/index.php

```diff
@@ -527,27 +527,22 @@
 
     // Update system wide smtp relay
     if (empty($_SESSION['error_msg'])) {
-        if (isset($_POST['v_smtp_relay']) && (!empty($_POST['v_smtp_relay_host'])) && (!empty($_POST['v_smtp_relay_user']))) {
+        if (isset($_POST['v_smtp_relay']) && !empty($_POST['v_smtp_relay_host'])) {
             if (($_POST['v_smtp_relay_host'] != $v_smtp_relay_host) ||
                      ($_POST['v_smtp_relay_user'] != $v_smtp_relay_user) ||
-                     ($_POST['v_smtp_relay_port'] != $v_smtp_relay_port) ||
```

Comment:
Port changes should still trigger v-add-sys-smtp-relay so is shouldn't be removed

Reply:
See next line as well, this change is to remove the requirement for empty smtp_relay_pass, the smtp_relay_port is still on the same line